### DISCUSSION TITLE
GTIN Exemption for PutListingItem

### DISCUSSION THREAD

**Discussion by 200317314** [abuzuhri/Amazon-SP-API-CSharp#722] [Q&A]
I am adding a parent and child to via putlistingitem, I have the GTIN exemption on amazon for Generic Home goods, I am submitting a Home product type
I am passing in this attribute "supplier_declared_has_product_identifier_exemption" and when i go to the product on amazon page it shows that the external id is missing, then I need to check the "I dont have a gtin" and it will do the exception.

How can I do this from my code?

**Comment by optitsolutions** [comment]
Sehr geehrte Damen und Herren,

bis inklusive 03.04.2024 bin ich nicht im B++ro und lese meine Mails nur unregelm+ñ+ig.

Bitte wenden Sie sich in dringenden F+ñllen an das B++ro unter ***@***.***

Ihre Mail wird nicht automatisch weitergeleitet.




Mit freundlichen Gr+++en

Thorsten Altmann






I am adding a parent and child to via putlistingitem, I have the GTIN exemption on amazon for Generic Home goods, I am submitting a Home product type
I am passing in this attribute "supplier_declared_has_product_identifier_exemption" and when i go to the product on amazon page it shows that the external id is missing, then I need to check the "I dont have a gtin" and it will do the exception.

How can I do this from my code?

-- 
Reply to this email directly or view it on GitHub:
https://github.com/abuzuhri/Amazon-SP-API-CSharp/discussions/722
